### ISSUE TITLE
Keep the build folder for cmake output?

### ISSUE THREAD

**Issue by francisdb** [vpinball/vpinball#1802]
**Is your feature request related to a problem? Please describe.**

Somewhere during 10.8.1 refactorings a `build` folder was introduced in repository root which contains windows build related files. However that folder was also already used in the build instructions for standalone as the build output folder.

_Is it standard practice to output to the `build` folder when using cmake?_ @jsm174 

The fact that output goes in this folder and that it is also in git is quite annoying. A git status will show you this:

```
➜  vpinball git:(build/asahi-linux) ✗ git status
On branch build/asahi-linux
Your branch is up to date with 'francisdb/build/asahi-linux'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        build/CMakeCache.txt
        build/CMakeFiles/
        build/Makefile
        build/VPinballX_BGFX
        build/VPinballX_GL
        build/assets/
        build/cmake_install.cmake
        build/docs/
        build/flexdmd/
        build/libSDL2-2.0.so
        ...
```

**Describe the solution you'd like**

Can we rename the repository build folder to something else?

**Describe alternatives you've considered**

* Adding git excludes/includes for the build folder, might be annoying to manage.
* Switching to a different output folder in the standalone build instructions.

**Comment by jsm174** [comment]
fwiw, out of the box, the official cmake tools and cmake extension packs from microsoft for visual studio code write to `build`

**Comment by francisdb** [comment]
fixed in #1838
